### PR TITLE
Jenkins: Add trigger for upgrade tests

### PR DESCRIPTION
**Pull request by Jaakko-Os**

### CODE REVIEW

**Comment by furkatgofurov7** [comment]
Did not see @fmuyassarov comment, I was talking also about those files

File: jenkins/scripts/integration_test.sh [hunk 1]
```diff
@@ -43,7 +43,7 @@ VM_TIMELABEL="${VM_TIMELABEL:-$(date '+%Y%m%d%H%M%S')}"
 TEST_EXECUTER_VM_NAME="${TEST_EXECUTER_VM_NAME:-ci-test-vm-${VM_TIMELABEL}}"
 TEST_EXECUTER_PORT_NAME="${TEST_EXECUTER_PORT_NAME:-${TEST_EXECUTER_VM_NAME}-int-port}"
 
-if [ "${TESTS_FOR}" == "feature_tests" ]
+if [ "${TESTS_FOR}" == "feature_tests" ] || [ "${TESTS_FOR}" == "feature_tests_upgrade" ]
```
Comment (maelk) [inline jenkins/scripts/integration_test.sh:46]: you could catch both by putting a wildcard in the first match

File: jenkins/jobs/integration_tests.pipeline [hunk 2]
```diff
@@ -6,7 +6,7 @@ def CLEAN_TIMEOUT = 600
 def TIMEOUT = 5400
 
 if (env.TESTS_FOR) {
-  if ("${TESTS_FOR}" == "feature_tests") {
+  if ("${TESTS_FOR}" == "feature_tests" || "${TESTS_FOR}" == "feature_tests_upgrade") {
```
Comment (maelk) [inline jenkins/jobs/integration_tests.pipeline:9]: you could catch both by putting a wildcard in the first match

File: jenkins/scripts/files/run_integration_tests.sh [hunk 3]
```diff
@@ -123,6 +123,9 @@ fi
 if [ "${TESTS_FOR}" == "feature_tests" ]
 then
   make feature_tests
```
Comment (maelk) [inline jenkins/scripts/files/run_integration_tests.sh:125]: what about catching both and running `make ${TESTS_FOR}`

File: jenkins/jobs/integration_tests.pipeline (hunk 2 above)
Comment (Jaakko-Os) [reply jenkins/jobs/integration_tests.pipeline]: done

File: jenkins/scripts/files/run_integration_tests.sh (hunk 3 above)
Comment (Jaakko-Os) [reply jenkins/scripts/files/run_integration_tests.sh]: done

File: jenkins/scripts/integration_test.sh (hunk 1 above)
Comment (Jaakko-Os) [reply jenkins/scripts/integration_test.sh]: done

**Pull request merged**